### DISCUSSION TITLE
Dividing lines

### DISCUSSION THREAD

**Discussion by Sh0ckwav3** [manuelbl/SwissQRBill.NET#58] [Q&A]
Hallo Manuel,

Vielen Dank für die super Library!
Zwei kleine Fragen:
1. Ist es möglich die Trennlinie zwischen dem Empfangsschein und dem QR Teil zu entfernen?
2. Kann man eine solche Trennlinie automatisch einfügen lassen über dem Zahlschein (Horizontal), ich habe gesehen es gibt extra ein Format dafür, aber ich habe die Option nicht gefunden.

Liebe Grüsse

-------------------------------------------------------------------------------------------------------------------------------------------

Hello Manuel,

Thank you for the great library!
Two small questions:
1. is it possible to remove the separator line between the receipt and the QR part?
2. is it possible to have such a separator line automatically inserted above the payment slip (horizontally), I saw there is an extra format for it, but I didn't find the option.

Kind regards

**Comment by manuelbl** [comment]
Es ist einerseits möglich, die Linienart (ausgezogen, gestrichelt, mit/ohne Schere, oder ganz ohne Linien) zu steuern:

```
bill.Format.SeparatorType = SeparatorType.DashedLineWithScissors;
```

Siehe auch https://codecrete.net/SwissQRBill.NET/api/Codecrete.SwissQRBill.Generator.SeparatorType.html

Andererseits beeinflusst das Output-Grösse die Linien: Wenn das Grösse QrBillOnly gewählt wird, wird die horizontale Linie weggelassen.

Siehe auch https://codecrete.net/SwissQRBill.NET/api/Codecrete.SwissQRBill.Generator.OutputSize.html und https://github.com/manuelbl/SwissQRBill/wiki/Output-Sizes

Falls die Anforderung ist, nur die horizontale, aber nicht die vertikale Linie zu zeichnen, so ist das leider nicht möglich.
